corehq/project_limits/tests/test_gauge.py

```diff
@@ -1,212 +1,212 @@
-from unittest.mock import patch
+# from unittest.mock import patch
```

Comment:
Commenting this file out was necessary for a reason on my local machine. I accidentally committed it but put it back in the next commit.